app/models/concerns/fae/trackable.rb

```diff
@@ -106,5 +112,13 @@ def update_parent
       end
     end
 
+    def tracker_parent_attrs
```

Comment:
@jasonfine I refactored this method so we don't need to pass around so many arguments

https://github.com/wearefine/fae/pull/144/commits/37a10ed6617ea0b4d28b1dae171d173ec6115f1e